app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(item["name"], item["age"]) for item in people]
+    for item in people:
```

Comment:
use a more descriptive variable name than `item` to enhance code readability.By using `person` as the variable name,